pg-index-health/src/test/java/io/github/mfvanek/pg/common/maintenance/DatabaseChecksTest.java

```diff
@@ -74,7 +74,7 @@ void onEmptyDatabaseCheckShouldReturnNothing(@Nonnull final Diagnostic diagnosti
     }
 
     @ParameterizedTest
-    @EnumSource(value = Diagnostic.class, mode = EnumSource.Mode.EXCLUDE, names = {"BLOATED_INDEXES", "BLOATED_TABLES", "FOREIGN_KEYS_WITHOUT_INDEX"})
+    @EnumSource(value = Diagnostic.class, mode = EnumSource.Mode.EXCLUDE, names = {"BLOATED_INDEXES", "BLOATED_TABLES", "FOREIGN_KEYS_WITHOUT_INDEX", "SEQUENCE_OVERFLOW"})
```

Comment:
Why SEQUENCE_OVERFLOW has been excluded?

Reply:
For Sequence Overflow, isEmpty was not included because it did not work at all.

Reply:
Reducing the threshold to 10.0 solved it.